ravendb/resource_ravendb_server_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
 	"github.com/ravendb/ravendb-go-client"
 	"github.com/ravendb/ravendb-go-client/serverwide/operations"
+	internal_operations "github.com/ravendb/terraform-provider-ravendb/operations"
```

Comment:
Not sure how it's related but line 12 is starting with `internal_operations` as well?